packages/ecma402-abstract/tests/FormatNumericRange.test.ts

```diff
@@ -0,0 +1,33 @@
+import {FormatNumericRange} from '../NumberFormat/FormatNumericRange'
+import {getInternalSlots} from './utils'
+
+describe('FormatNumericRange', () => {
+  const numberFormat: Intl.NumberFormat = new Intl.NumberFormat('it')
+
+  it('correctly return numeric range', () => {
+    const result = FormatNumericRange(numberFormat, 3, 10, {getInternalSlots})
+
+    expect(result).toBe('3–10')
+  })
+
+  it('correctly return numeric range with single negative number', () => {
+    const result = FormatNumericRange(numberFormat, -3, 10, {getInternalSlots})
+
+    expect(result).toBe('-3–10')
+  })
+
+  it('correctly return numeric range with both negative numbers', () => {
+    const result = FormatNumericRange(numberFormat, -10, -3, {getInternalSlots})
+
+    expect(result).toBe('-10–-3')
```

Comment:
this seems wrong, looks like ILND is not enough because it's also dependent on the value.
Currently Chrome `formatRange(3, 10)` yields `3-10` but `formatRange(-3, -10)` yields `-3 - -10` so the spec needs to clarify this IMO.
`CollapseNumberRange` does not seem to cover this IMO
Thoughts @sffc?

Reply:
Good question. The UTS 35 spec is here:

http://www.unicode.org/reports/tr35/tr35-numbers.html#8-number-range-formatting

ECMA-402 does not require adherence to UTS 35, but it's a decent reference for what to do here. Regarding spacing around the range separator, it says:

> To determine whether to add spacing, the currently recommended heuristic is:
>
> 1. If the lower string ends with a character other than a digit, or if the upper string begins with a character other than a digit.
> 2. If the range pattern does not contain a character having the White_Space binary Unicode property after the `{0}` or before the `{1}` placeholders.

If both those properties are true, then add spaces. In this case, the first character of the lower string starts with a non-number character ("-"), so adding spaces is justified.

Reply:
Do you think we should port this to ecma-402 or handle it in some way? Currently as the spec is written the range separator is a constant that only varies based on ILND factors and not the value itself, so inserting whitespace technically not spec-compliant.

I think a simple test262 of creating a blank Intl.NumberFormat() and formatRange on these 2 test cases I mentioned above resulting in 2 range separator would make Chrome not compliant right?

Reply:
I think this is handled in `CollapseNumberRange`, which is intended to give the implementation a lot of leeway to modify the output string, but it wouldn't hurt to clarify that.

Reply:
https://github.com/tc39/proposal-intl-numberformat-v3/issues/113

Reply:
![image](https://user-images.githubusercontent.com/198255/198049031-11bdd8dd-63d3-40e5-92bf-9d5dabbf8e82.png)

Reply:
@sffc it does seem like a spec issue with this behavior?

Reply:
The range check was removed in July: https://github.com/tc39/proposal-intl-numberformat-v3/pull/100

Is the HTML version of the spec out of date? I'll fix it.

Reply:
I've removed all the code related to that part.